packages/nextjs/pages/index.tsx

```diff
@@ -48,76 +48,48 @@ const Home: NextPage<{
+      {/* Start Building on Ethereum */}
+      <div className="bg-white pb-16" id="start-building-on-ethereum">
+        <div className="container max-w-[90%] lg:max-w-6xl m-auto py-16 lg:py-28 lg:px-12 flex flex-col items-center lg:items-start">
+          <div className="bg-[url(/assets/spaceship.png)] bg-no-repeat bg-right-top lg:max-w-[580px] bg-[length:85px] lg:bg-[length:100px] max-w-[500px] pb-10">
             <div className="mt-6 lg:mt-0">
               <h2 className="text-3xl lg:text-5xl text-center lg:text-left lg:max-w-lg">
-                Already have Solidity skills?
+                Start building <br /> on Ethereum
               </h2>
               <p className="text-center lg:text-left lg:w-3/4">
-                Test your knowledge by building real-world, complex apps.
+                Start your journey with Speedrun Ethereum, then continue learning in our Batches program and opt to our
+                small grants to improve your skills!
               </p>
             </div>
           </div>
           {/* Cards container */}
           <div className="flex gap-8 flex-wrap lg:flex-nowrap justify-center lg:max-w-5xl">
             {/* Card */}
             <Card
-              num={4}
-              name="Build a DEX"
+              name="Speedrun Ethereum"
               src="/assets/chall-dex.png"
-              description="💵 Build an exchange that swaps ETH to tokens and tokens to ETH."
+              description="Learn the most important concepts and level up your skills."
               link="https://github.com/scaffold-eth/se-2-challenges/tree/challenge-4-dex"
             />
             <Card
-              num={5}
-              name="State Channels"
-              src="/assets/chall-state.png"
-              description="🐌 The Ethereum blockchain has great decentralization & security properties. These properties come at a price!"
-              link="https://github.com/scaffold-eth/se-2-challenges/tree/challenge-5-state-channels"
+              name="Join the batches"
+              src="/assets/batches-card.png"
+              description="Continue learning in our Batches monthly program for up-and-coming devs."
+              link="https://buidlguidl.com/batches"
             />
             <Card
-              num={6}
-              name="Multisig Wallet"
-              src="/assets/chall-multisig.png"
-              description="👩‍👩‍👧‍👧 Using a smart contract as a wallet we can secure assets by requiring multiple accounts to 'vote' on transactions."
-              link="https://github.com/scaffold-eth/se-2-challenges/tree/challenge-6-multisig"
+              name="Small Grants"
+              src="/assets/bg-grants-card.png"
+              description="BuidlGuidl can sponsor up to 1 ETH to build your idea. Funding meaningful projects across the ecosystem."
+              link="https://grants-bg.vercel.app"
```

Comment:
We can point to: https://grants.buidlguidl.com/

Reply:
Did you push it to this branch?

Reply:
Yes! You can find it above these comments in this thread, right before your commit

Reply:
Oh wait! I read it wrong 😅 I thought we were only talking about batches all the time, sorry! 

Just updated it now in a commit 👌🏻